scrapy/linkextractors/lxmlhtml.py

```diff
@@ -57,8 +57,9 @@ def _extract_links(self, selector, response_url, response_encoding, base_url):
                 url = url.encode(response_encoding)
             # to fix relative links after process_value
             url = urljoin(response_url, url)
+            rel = el.get('rel')
             link = Link(url, _collect_string_content(el) or u'',
-                nofollow=True if el.get('rel') == 'nofollow' else False)
+                nofollow=True if rel is not None and 'nofollow' in rel.split() else False)
```

Comment:
I believe that utility functions will be more flexible. Would putting it in new package `scrapy.utils.link` sufficient?